### DISCUSSION TITLE
Problem CSS background since MySQL 5.7

### DISCUSSION THREAD

**Discussion by alex69270** [mpdf/mpdf#1640] [Q&A]
Hello, 

I use mpdf 6.0 and since I changed MySQL from 5.6 to 5.7, the background used in my css file doesn't work anymore.
`background-image: url("./img/image.jpg");`

I tested on another FTP with MySQL 5.6 and it works well.

Maybe you know why ?

Thanks for your help !

**Comment by ssieb** [comment]
mpdf doesn't use mysql, so that's not directly relevant.  And how does an FTP factor in here?  You need to provide a lot more details if you want help.
